lib/jekyll/version.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module Jekyll
-  VERSION = "3.8.2".freeze
+  VERSION = "3.8.2"
```

Comment:
![Let It Go](https://user-images.githubusercontent.com/251545/40795210-f3fe16a0-64c6-11e8-973e-0ab42b9a61ca.gif)